scaffoldmaker/meshtypes/meshtype_3d_sphereshell1.py

```diff
@@ -311,7 +324,10 @@ def generateBaseMesh(region, options):
                 for e1 in range(elementsCountAround):
                     va = e1
                     vb = (e1 + 1)%elementsCountAround
-                    eft1 = tricubichermite.createEftShellApexBottom(va*100, vb*100)
+                    if useCubicHermite:
```

Comment:
As per previous comment, with eftfactory, using duck typing can replace this with just:

    eft1 = eftfactory.createEftShellPoleBottom(va*100, vb*100)